app/controllers/provider/admin/authentication_providers_controller.rb

```diff
@@ -98,7 +98,8 @@ def authorize_authentication_provider(action = action_name)
                      username_key trust_email].freeze
 
   def update_params
-    params.require(:authentication_provider).permit(UPDATE_PARAMS)
+    defaults = { branding_state_event: 'custom_brand' }
```

Comment:
Can we make it a constant of the class instead? E.g.

```ruby
class Provider::Admin::AuthenticationProvidersController < FrontendController
  (...)

  UPDATE_PARAM_DEFAULTS = { branding_state_event: 'custom_brand' }.freeze

  def update_params
    params.require(:authentication_provider).permit(UPDATE_PARAMS).reverse_merge(UPDATE_PARAM_DEFAULTS)
  end

  (...)
end
```

Reply:
_Question:_ does it have to be a default on create as well? If so, should the default be better defined this at the level of the model maybe?